public/form/js/function.js

```diff
@@ -0,0 +1,8 @@
+$( "#open_menu" ).click(function() {
```

Comment:
We don't need jQuery for that, and we would probably set a class. Let me explain you why (hint: autoprefixed rules).